tests/Application/public/index.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+declare(strict_types=1);
+
+use Tests\Sylius\PriceHistoryPlugin\Application\Kernel;
+use Symfony\Component\ErrorHandler\Debug;
+use Symfony\Component\HttpFoundation\Request;
+
+require dirname(__DIR__) . '/config/bootstrap.php';
```

Comment:
We can use SF Runtime, eg. https://github.com/Sylius/Sylius-Standard/commit/2b99bdbf8cc6e715ac907ef7306120cc3cbdfabb